includes/wc-core-functions.php

```diff
@@ -1491,7 +1491,7 @@ function wc_get_shipping_method_count( $include_legacy = false ) {
 		return absint( $transient_value['value'] );
 	}
 
-	$method_count = absint( $wpdb->get_var( "SELECT COUNT(*) FROM {$wpdb->prefix}woocommerce_shipping_zone_methods" ) );
+	$method_count = absint( $wpdb->get_var( "SELECT COUNT(*) FROM {$wpdb->prefix}woocommerce_shipping_zone_methods WHERE is_enabled=true" ) );
```

Comment:
The `is_enabled` column used `tinyint` -- should this be `WHERE is_enabled=1`?

Reply:
Changed in 58a90f95c5fc0e8148eccbb822624ec7899677b8. :+1: